react-hash-routing/client/pages/not-found.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const styles = {
+  pageContent: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    minHeight: 'calc(100vh - 3.5rem)'
+  }
+};
+
+export default function NotFound(props) {
+  return (
+    <div style={styles.pageContent}>
+      <div className="row">
+        <div className="col text-center mb-5">
+          <h3>
+            Uh oh, we could not find the page you were looking for!
+          </h3>
+          <p className="text-muted">
+            {/* this anchor should go back to the catalog at '#' */}
+            <a href="">Return to the catalog</a>
```

Comment:
There is one here too.

Reply:
<img width="752" alt="Screen Shot 2022-09-07 at 3 07 09 PM" src="https://user-images.githubusercontent.com/106217355/188992196-dd56402f-efd7-44cd-8b91-0e0b5e438fc5.png">